### PR TITLE
Remove spelling terms.

### PR DESCRIPTION
**Pull request by mk46**
Signed-off-by: Manish Kumar <manish.kumar1@india.nec.com>

Commit Message: Remove spelling terms.
Additional Description: Cleaning `tools/spelling/spelling_dictionary.txt` which are single-use keywords. 
Risk Level:
Testing:
Docs Changes:
Release Notes:
Platform Specific Features:
Includes a partial fix to #12807

### CODE REVIEW

**Review by ggreenway — DISMISSED** [review]
LGTM. Thanks!

**Comment by ggreenway** [comment]
check-format failed:

```

Format check failed, try apply following patch to fix:
diff --git a/source/server/worker_impl.cc b/source/server/worker_impl.cc
index fa9212d84..f59da83f5 100644
--- a/source/server/worker_impl.cc
+++ b/source/server/worker_impl.cc
@@ -40,8 +40,8 @@ void WorkerImpl::addListener(absl::optional<uint64_t> overridden_listener,
                              Network::ListenerConfig& listener, AddListenerCompletion completion) {
   // All listener additions happen via post. However, we must deal with the case where the listener
   // can not be created on the worker. There is a race condition where 2 processes can successfully
-  // bind to an address, but then fail to listen() with `EADDRINUSE`. During initial startup, we want
-  // to surface this.
+  // bind to an address, but then fail to listen() with `EADDRINUSE`. During initial startup, we
+  // want to surface this.
   dispatcher_->post([this, overridden_listener, &listener, completion]() -> void {
     try {
       handler_->addListener(overridden_listener, listener);
```

/wait

File: source/extensions/transport_sockets/tls/context_impl.h [hunk 1]
```diff
@@ -169,7 +169,7 @@ class ContextImpl : public virtual Envoy::Ssl::Context {
                      const Ocsp::OcspResponseWrapper* ocsp_response) const;
 
   struct TlsContext {
-    // Each certificate specified for the context has its own SSL_CTX. SSL_CTXs
+    // Each certificate specified for the context has its own SSL_CTX. `SSL_CTXs`
```
Comment (antoniovicente) [inline source/extensions/transport_sockets/tls/context_impl.h:172]: Drive by: SSL_CTX being in the dictionary but SSL_CTXs requiring ``'s around it seems a bit strange.

**Comment by mk46** [comment]
Fixed check-format.

File: source/extensions/transport_sockets/tls/context_impl.h (hunk 1 above)
Comment (mk46) [reply source/extensions/transport_sockets/tls/context_impl.h:172]: While removing CTXs from the dictionary SSL_CTXs requires ``'s around it.

Comment (ggreenway) [reply source/extensions/transport_sockets/tls/context_impl.h:172]: Looks a little weird, but I'm guessing it'll get cleaner as more of the dictionary is cleaned out.

**Review by ggreenway — APPROVED** [review]

**Pull request merged**